recipes/notify.rb

```diff
@@ -83,6 +84,16 @@
       run_locally "cd #{strategy.local_cache_path}; git push -f #{repository} HEAD:refs/heads/deployed-to-#{ENV['ORGANISATION']}"
     end
 
+    task :docker, only: { primary: true } do
+      if ENV['DOCKER_HUB_USERNAME'] && ENV['DOCKER_HUB_PASSWORD']
+        repo = "govuk/#{application}"
+
+        pusher = DockerTagPusher.new(ENV['DOCKER_HUB_USERNAME'], ENV['DOCKER_HUB_PASSWORD'])
+        manifest = pusher.get_manifest(repo, branch)
+        pusher.put_manifest(repo, manifest, "deployed-to-#{ENV['ORGANISATION']}")
```

Comment:
I wonder if you might want to just have a single method call on the tag pusher that does both of these for you like `tag_branch`